README.md

```diff
@@ -57,6 +85,21 @@ Welcome to the Cincinnati Women Who Code repository, a Gatsby project that showc
   <img alt="WWC" src="src/images/prReviewers.png" width="250" />
   You are now done and your pull request will be code reviewed. You will be notified regarding any feedback and when it has been approved.
   
+## 🎶	 Keep your fork in sync 
+  
+  It is very important to keep your fork in sync with the original repo.  It is recommended that you sync prior to starting a new branch and also prior to creating pull request.
+  Navigate to your main branch and execute this:
+ ```shell
+ git fetch upstream
+ ```
+Merge the changes from the upstream/main into your local main branch
+```shell
+git merge upstream/main
+```
+Push those changes to your fork 
+```shell
+git push origin main
+```
```

Comment:
This is pretty low on the readMe, but I guess we'll just have to reiterate to read the entire readMe during the presentation so others remember to keep their forks in sync